### PR TITLE
Add Github templates

### LINKED ISSUE #6: Add minimal instructions for contribution guidelines and issue creation
For development purposes, it would be great to create a `CONTRIBUTING.md` file and issues/PRs templates, as it allows developers and testers to easily create issues and PRs on the project.

### PR DESCRIPTION
**Pull request by Dynnammo**
Add minimal files to ease issues and PRs creation.
Closes #6

### CODE REVIEW

**Comment by mawoka-myblock** [comment]
Great, thank you! I've wrote some comments on your commits, please have a look at them.

**Review by mawoka-myblock — APPROVED** [review]
Please add a

**Comment by Dynnammo** [comment]
> Please add a

?

**Review by mawoka-myblock — CHANGES_REQUESTED** [review]

**Comment by mawoka-myblock** [comment]
> > Please add a
> 
> ?

Sorry, clicked too fast 😅

**Comment by mawoka-myblock** [comment]
Why did you delete all the great templates?

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+**Stacktrace**
+If applicable, add the error stacktrace to help explain your problem.
+
+**Extra data (please complete the following information):**
+ - Device [e.g. iPhone6, Desktop]:
+ - Device OS [e.g. iOS8.1, Windows 10]:
+ - Browser [e.g. Chrome, Firefox, Safari]:
+ - ~~Classquiz Version [e.g. 0.10]:~~  :warning:** No versioning for now** :warning:
+ - Classquiz installation [e.g. classquiz.mawoka.eu]:
+ - Commit SHA [e.g : ed35c5c69019e7cd6331ea438519f53b8bce70cf]:
```
Comment (Dynnammo) [inline .github/ISSUE_TEMPLATE/Bug_report.md:33]: Commit SHA asked here

**Review by mawoka-myblock — APPROVED** [review]

**Comment by Dynnammo** [comment]
:fire:

**Pull request merged**